test/relay_client_test.js

```diff
@@ -146,7 +150,8 @@ contract('RelayClient', function (accounts) {
         from: gasLess,
         to: to,
         txfee: 12,
-        gas_limit: 1000000
+        gas_limit: 1000000,
```

Comment:
I saw elsewhere that you changed it to gasLimit

Reply:
###4 as well